From patchwork Tue Sep 18 17:43:20 2018
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
X-Patchwork-Submitter: Peng Hao <peng.hao2@zte.com.cn>
X-Patchwork-Id: 10603879
Return-Path: <kvm-owner@kernel.org>
Received: from mail.wl.linuxfoundation.org (pdx-wl-mail.web.codeaurora.org
 [172.30.200.125])
	by pdx-korg-patchwork-2.web.codeaurora.org (Postfix) with ESMTP id C3A22157B
	for <patchwork-kvm@patchwork.kernel.org>;
 Tue, 18 Sep 2018 09:30:51 +0000 (UTC)
Received: from mail.wl.linuxfoundation.org (localhost [127.0.0.1])
	by mail.wl.linuxfoundation.org (Postfix) with ESMTP id B1B6129FD0
	for <patchwork-kvm@patchwork.kernel.org>;
 Tue, 18 Sep 2018 09:30:51 +0000 (UTC)
Received: by mail.wl.linuxfoundation.org (Postfix, from userid 486)
	id A37FF29FE3; Tue, 18 Sep 2018 09:30:51 +0000 (UTC)
X-Spam-Checker-Version: SpamAssassin 3.3.1 (2010-03-16) on
	pdx-wl-mail.web.codeaurora.org
X-Spam-Level: 
X-Spam-Status: No, score=-7.9 required=2.0 tests=BAYES_00,MAILING_LIST_MULTI,
	RCVD_IN_DNSWL_HI autolearn=ham version=3.3.1
Received: from vger.kernel.org (vger.kernel.org [209.132.180.67])
	by mail.wl.linuxfoundation.org (Postfix) with ESMTP id 277B529FD0
	for <patchwork-kvm@patchwork.kernel.org>;
 Tue, 18 Sep 2018 09:30:51 +0000 (UTC)
Received: (majordomo@vger.kernel.org) by vger.kernel.org via listexpand
        id S1728645AbeIRPCe (ORCPT
        <rfc822;patchwork-kvm@patchwork.kernel.org>);
        Tue, 18 Sep 2018 11:02:34 -0400
Received: from mxhk.zte.com.cn ([63.217.80.70]:55142 "EHLO mxhk.zte.com.cn"
        rhost-flags-OK-OK-OK-OK) by vger.kernel.org with ESMTP
        id S1726768AbeIRPCe (ORCPT <rfc822;kvm@vger.kernel.org>);
        Tue, 18 Sep 2018 11:02:34 -0400
Received: from mse01.zte.com.cn (unknown [10.30.3.20])
        by Forcepoint Email with ESMTPS id 838093A91DCED8280A4D;
        Tue, 18 Sep 2018 17:30:46 +0800 (CST)
Received: from notes_smtp.zte.com.cn ([10.30.1.239])
        by mse01.zte.com.cn with ESMTP id w8I9U4SE080643;
        Tue, 18 Sep 2018 17:30:04 +0800 (GMT-8)
        (envelope-from peng.hao2@zte.com.cn)
Received: from localhost.localdomain.localdomain ([10.74.120.59])
          by szsmtp06.zte.com.cn (Lotus Domino Release 8.5.3FP6)
          with ESMTP id 2018091817300617-4737 ;
          Tue, 18 Sep 2018 17:30:06 +0800
From: Peng Hao <peng.hao2@zte.com.cn>
To: pbonzini@redhat.com, rkrcmar@redhat.com
Cc: kvm@vger.kernel.org, Peng Hao <peng.hao2@zte.com.cn>
Subject: [PATCH]  kvm-unit-test: fix script output abnormal
Date: Wed, 19 Sep 2018 01:43:20 +0800
Message-Id: <1537292600-29202-1-git-send-email-peng.hao2@zte.com.cn>
X-Mailer: git-send-email 1.8.3.1
X-MIMETrack: Itemize by SMTP Server on SZSMTP06/server/zte_ltd(Release
 8.5.3FP6|November
 21, 2013) at 2018-09-18 17:30:06,
        Serialize by Router on notes_smtp/zte_ltd(Release 9.0.1FP7|August  17,
 2016) at
 2018-09-18 17:30:00,
        Serialize complete at 2018-09-18 17:30:00
X-MAIL: mse01.zte.com.cn w8I9U4SE080643
Sender: kvm-owner@vger.kernel.org
Precedence: bulk
List-ID: <kvm.vger.kernel.org>
X-Mailing-List: kvm@vger.kernel.org
X-Virus-Scanned: ClamAV using ClamSMTP

./run_tests.sh
)ASS apic-split (50 tests;
)ASS ioapic-split (19 tests;
)ASS apic (50 tests;
)ASS ioapic (19 tests;
)ASS smptest (1 tests;
)ASS smptest3 (1 tests;

modify sed statement in scripts/runtime.bash, we can see normal
output.

Signed-off-by: Peng Hao <peng.hao2@zte.com.cn>
---
 scripts/runtime.bash | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/runtime.bash b/scripts/runtime.bash
index a31ae91..37f8ebc 100644
--- a/scripts/runtime.bash
+++ b/scripts/runtime.bash
@@ -8,7 +8,7 @@ FAIL() { echo -ne "\e[31mFAIL\e[0m"; }
 
 extract_summary()
 {
-    tail -3 | grep '^SUMMARY: ' | sed 's/^SUMMARY: /(/;s/$/)/'
+    tail -3 | grep '^SUMMARY: ' | sed 's/^SUMMARY: /(/;s/\r$/)/'
 }
 
 # We assume that QEMU is going to work if it tried to load the kernel
